src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private Colors color;
+
+    @Override
+    public String toString() {
+        return "Color: " + color + " Number: " + number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(Colors color) {
+        this.color = color;
+    }
+
+    public Colors getColor() {
+        return color;
+    }
```

Comment:
Getters: you never used them. Remove it please.
Setters: Imho it's better to assign values to `number` and `color` during creating an object (using Constructor)